yarn.lock

```diff
@@ -1268,6 +1268,87 @@
   resolved "https://registry.npmjs.org/@juggle/resize-observer/-/resize-observer-3.3.1.tgz#b50a781709c81e10701004214340f25475a171a0"
   integrity sha512-zMM9Ds+SawiUkakS7y94Ymqx+S0ORzpG3frZirN3l+UlXUmSUR7hF4wxCVqW+ei94JzV5kt0uXBcoOEAuiydrw==
 
+"@jupyter-widgets/base-manager@^1.0.0-rc.1":
```

Comment:
this is concerning, as it may complicate e.g. `ipywidgets` injecting their own federated extension to overload this one. We may need to add some way for apps to specify federated extensions.